Wrap mysql credentials for cli use in quotes

JonasPf opened the pull request:
This allows one to use reserved shell characters like '*' in
username and passwords. Without this change the shell would
interpret and expand those characters.

skale-cla commented:
[![CLA assistant check](http://cla.skale.network/pull/badge/signed)](http://cla.skale.network/skalenetwork/skale-node-cli?pullRequest=317) <br/>All committers have signed the CLA.

dmytrotkk reviewed (approved):
Thanks for your contribution 🤗

gannakulikova commented:
Hi @JonasPf !
thank you very much for contribution and the fix! The required step in SKALE CI process is to get the commits signed, could you please have a look?
Some docs for your reference: https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits

JonasPf committed:
Wrap mysql credentials for cli use in quotes

This allows one to use reserved shell characters like '*' in
username and passwords. Without this change the shell would
interpret and expand those characters.

JonasPf commented:
> Hi @JonasPf !
> thank you very much for contribution and the fix! The required step in SKALE CI process is to get the commits signed, could you please have a look?
> Some docs for your reference: https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits

should be fixed now, thanks!
